app/models/User.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] `:confirmable` needs enabled in the user model (`user.rb` file).

Reply:
@GzimAsani after adding the above suggestion it immediately followed with an error do you have any suggestions on how to solve the problem